main/src/main/scala/sbt/internal/CommandExchange.scala

```diff
@@ -154,6 +155,11 @@ private[sbt] final class CommandExchange {
           case Some(Success(_)) =>
             // rememeber to shutdown only when the server comes up
             server = Some(x)
+          case Some(Failure(e: AlreadyRunningException)) =>
+            s.log.warn("Is another instance of sbt is running on this build?")
```

Comment:
I don't think I like the warning to be in question form. Feels too unsure. How about just "Another instance of sbt is running on this build".

Reply:
I am unsure. Is this better?

"sbt server could not start likely because there's another instance of sbt running on this build."

Reply:
I'd be happy with that if it didn't have the "likely".

Then if down the line we find there's another reason why `AlreadyRunningException` is matched here that isn't because of multiple instances we can make the necessary changes.